src/util/ClientServiceUtility.java

```diff
@@ -0,0 +1,7 @@
+package util;
+
+public final class ClientServiceUtility {
+  public static String createCode(Long currentId,String str) {
+    return String.format("%s%07d", str,currentId);
```

Comment:
Why did you add another argument? Why not just `String.format("C_%s", code)`?

Reply:
solved